modules/installation-creating-aws-worker.adoc

```diff
@@ -104,37 +104,12 @@ directory. This value is the long string with the format
 the CloudFormation template for the security group and roles.
 <15> The type of AWS instance to use for the control plane machines.
 <16> Allowed values:
-* `m4.large`
```

Comment:
@kalexand-rh please refer to values in upi/aws/cloudformation/06_cluster_worker_node.yaml in https://github.com/openshift/installer/pull/4327/files , thanks